Sources/MapboxNavigation/CarPlayNavigationViewControllerDelegate.swift

```diff
@@ -70,8 +70,8 @@ public extension CarPlayNavigationViewControllerDelegate {
     /**
      `UnimplementedLogging` prints a warning to standard output the first time this method is called.
      */
-    func carPlayNavigationViewControllerDidDismiss(_ carPlayNavigationViewController: CarPlayNavigationViewController,
```

Comment:
Oops, it was my typo and now it's treated as a breaking API change. I think it's okay to rename it here as it's internal anyway (but `CarPlayNavigationViewControllerDelegate` is public).